src/main/java/codeu/model/data/User.java

```diff
@@ -40,6 +41,7 @@ public User(UUID id, String name, String passwordHash, Instant creation) {
     this.creation = creation;
     aboutMe = "";
     isAdmin = false;
+
```

Comment:
It might be a good idea to set a value for `lastSeenNotificationsTimestamp` in the constructor. Perhaps it can be null that way we know the user is new/hasn't seen any notifications.

Reply:
+1.
You can also add a unit test for it, to make sure it gets initialized to null for a new user.

Reply:
Would it be better to set the time the creation of the user class? Instant has a function now() that sets the time at the current moment because we wouldn't want new users to see past notifications.

Reply:
I think this is actually more accurate, thanks for catching that. Yes we should probably use the current time as value for this field on new users.